components/eventing-controller/README.md

```diff
@@ -0,0 +1,112 @@
+# Eventing controller
+
+## Overview
+
+This component contains controllers for various CustomResourceDefinitions related to eventing in Kyma. The controller comes with 2 containers:
+- [controller](https://github.com/kubernetes-sigs/controller-runtime)
+- [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy)
+
+
+## Prerequisites
+- Install [ko](https://github.com/google/ko) which is used to build and deploy the controller during local development
+- Install [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) which is the base framework for this controller
+- Install [kustomize](https://github.com/kubernetes-sigs/kustomize) which lets customize raw, template-free YAML files during local development
+
+### Installation
+
+- To deploy the controller inside a cluster, make sure you have `ko` installed and configured according to the [usage instructions](https://github.com/google/ko#usage), then run:
+
+    ```sh
+    make deploy-local
+
+    ## To verify all the manifests after the processing by Kustomize without applying to the cluster use make target deploy-local-dry-run    
+    make deploy-local-dry-run
+    ```
+
+## Usage 
+
+This section explains how to use the eventing-controller.
+
+- The eventing-controller comes with the following command line argument flags
+
+    | Flag    | Default Value | Description                                                                                   |
+    | ----------------------- | ------------- |---------------------------------------------------------------------------------------------- |
+    | metrics-addr            | :8080          | It's address the metric endpoint binds to.
```

Comment:
```suggestion
    | metrics-addr            | :8080          | Address the metric endpoint binds to.
```

Not sure what "it's address the metric endpoint binds to" means, if it's the Eventing Controller address, we can maybe write "Eventing Controller address the metric endpoint binds to"?